### DISCUSSION TITLE
Remove thanks page in favor of lightsail hosted site.

### DISCUSSION THREAD

**Discussion by yycholla** [cat-box/aws-foundry-ssl#37] [Q&A]
Is there any way for me to get rid of the thanks page and replace it with a lightsail domain? I want to be able to use one domain for most of my website needs and have been using subdomains in order to do so. Would love to have access to the main domain; however, I am not familiar enough with nginx to find the answer myself.

**Comment by cat-box** [comment]
I've never used lightsail but amazon lightsail's documentation indicates that you can. There is an option explained in [Template Parameters](https://github.com/cat-box/aws-foundry-ssl/wiki/CloudFormation-Template-Deployment#ssl-configuration-parameters) listed as "WebServerBool". Set that as **False** and then follow the instructions [here](https://lightsail.aws.amazon.com/ls/docs/en_us/articles/amazon-lightsail-using-route-53-to-point-a-domain-to-an-instance). 

I hope that helps, sorry for the late reply!
